.github/configs/labeler.yml

```diff
@@ -0,0 +1,30 @@
+Atlas:
+  - packages/**/*atlas*/**/*
+documentation:
+  - "*.md"
+Cross-Platform:
+  - packages/tools/pluggable-widgets-tools/**/*
+  - packages/tools/generator/**/*
+  - packages/modules/atlas-core/**/*
+  - .github/**/*
+  - .husky/**/*
+  - data/**/*
+  - scripts/**/*
+Hybrid:
+  - packages/jsActions/*-hybrid/**/*
+Native:
+  - packages/**/*-native/**/*
+  - packages/**/*_native*/**/*
+  - detox/**/*
+Web:
+  - packages/*-web/**/*
+  - packages/shared/charts/**/*
+  - packages/**/*-web-*/**/*
+  - packages/**/*_web*/**/*
+  - packages/modules/data-widgets/**/*
+  - configs/e2e/**/*
+PWA:
+  - packages/**/barcode-scanner-web/**/*
+  - packages/**/nanoflow-actions-web/**/*
+test:
+  - "*.spec.*"
```

Comment:
Does a `test` label add a lot of value? Basically every PR that touches a widget should touch a spec file in some way or another, both web and native. I don't mind having it around, but I also don't really see a lot of value in it + we didn't use it a lot either.

Reply:
Fixed